test/packages/parallel/oracle/data_stream/memory/_dev/test/system/test-memory-config.yml

```diff
@@ -0,0 +1,16 @@
+vars:
+  hosts:
+    - "oracle://sys:Oradoc_db1@elastic-package-service-oracle-1:1521/ORCLCDB.localdomain?sysdba=1"
+agent:
+  custom_script: |
+    apt-get update && apt-get -y install libaio1 wget unzip
+    mkdir -p /opt/oracle
+    cd /opt/oracle
+    wget https://download.oracle.com/otn_software/linux/instantclient/214000/instantclient-basic-linux.x64-21.4.0.0.0dbru.zip && unzip -o instantclient-basic-linux.x64-21.4.0.0.0dbru.zip
+    wget https://download.oracle.com/otn_software/linux/instantclient/217000/instantclient-sqlplus-linux.x64-21.7.0.0.0dbru.zip && unzip -o instantclient-sqlplus-linux.x64-21.7.0.0.0dbru.zip
+    echo /opt/oracle/instantclient_21_4 > /etc/ld.so.conf.d/oracle-instantclient.conf && ldconfig
+    cp /opt/oracle/instantclient_21_7/glogin.sql /opt/oracle/instantclient_21_7/libsqlplus.so /opt/oracle/instantclient_21_7/libsqlplusic.so /opt/oracle/instantclient_21_7/sqlplus /opt/oracle/instantclient_21_4/
+  pre_start_script: |
+    export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/oracle/instantclient_21_4"
+    export PATH="${PATH}:/opt/oracle/instantclient_21_4"
+    cd /opt/oracle/instantclient_21_4
```

Comment:
Tested locally without this pre-start script and it also worked. However, I'd prefer to keep it since it was set in the Dockerfile too here:
https://github.com/elastic/elastic-package/blob/e53417ac3d0066e5b2b2385e73219831a4fdb01d/test/packages/with-custom-agent/oracle/data_stream/memory/_dev/deploy/agent/Dockerfile#L13-L16

Reply:
Added a new test package under `test/packages/other` with a minimal configuration to use these scripts.

About the test package `oracle` , do you think it should be kept under `parallel` as an example for the migration ?